pkg/consts/consts.go

```diff
@@ -213,9 +213,6 @@ var (
 
 // load balancer
 const (
-	// TODO: After dual-stack is supported, all references should be updated and this variable is not needed.
-	DualstackSupported = false
```

Comment:
DualstackSupported was used in pip service annotation (pip name, pip prefix id) and getResourceByIPFamily (pip, lb rule).
For service annotations, since `ServiceAnnotationPIPNameDualStack` and `ServiceAnnotationPIPPrefixIDDualStack` are new introduced, it is safe to add them directly. Old service annotations are still supported.
For `getResourceByIPFamily`, I added TODO comments at `getLoadBalancerRuleName()`, `getPublicIPName`, etc. Also, `getPublicIPName` will directly use `getResourceByIPFamily` in the next PR.